Src/BootCamp.Chapter/StringOps.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public static class StringOps
+    {
+        public static string Encode(string password)
+        {
+            var temp = Encoding.Unicode.GetBytes(password);
+            var encodedPassword = new StringBuilder();
+            foreach (var character in temp)
+            {
+                encodedPassword.Append(character);
+            }
+            return encodedPassword.ToString();
+        }
+
+        public static bool IsValid(string input)
+        {
+            return !string.IsNullOrWhiteSpace(input) || !string.IsNullOrEmpty(input);
```

Comment:
probably want's to be `&&`? not null, not whitespace and not empty?

Reply:
I don;t know how could I mistake AND with OR and keep everything working.
I fixed in [2a4af80](https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/232/commits/2a4af80f11d86f38c8efd0a1debe8b0fd5ae911e)